### ISSUE TITLE
Typo in profiling exercise

### ISSUE THREAD

**Issue by rushtongarth** [Foundations-of-Applied-Mathematics/Labs#5]
There appears to be a typo in exercise 7 of Lab 08.  In particular, there is a snippet of code which reads:
```python
def matrix_power(A, n):
    """Compute A^n, the n-th power of the matrix A."""
    product = A.copy()
    temporary_array = np.empty_like(A[0])
    m = A.shape[0]
    for power in range(1, power):
        #...
```
I think that the for loop should be changed to:
```python
     for power in range(1, n):
```

This can also be found on [line 757](https://github.com/Foundations-of-Applied-Mathematics/Labs/blob/44b516429dc7dfff26aba4a71f03b965802c20da/PythonEssentials/Profiling/Profiling.tex#L757) in the `profiling.tex` file

**Comment by shanemcq18** [comment]
We've corrected this in our development version and the change will be merged in soon. Thanks for pointing this one out.

**Comment by shanemcq18** [comment]
Corrected in 52144d52c7c1e5c867b1df1d565d3f0ff90e0059.
